_docs/ios/1.142.0.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: page
+title: 1.142.0
+permalink: /docs/ios/1.142.0/
+parent: iOS
+nav_order: -142
+---
+
+## Added Portuguese language
+We added support for Portuguese language in our application.\
+![screenshot](/tedee-release-notes/docs/ios/assets/1.142.0-portuguese.png)
+
+## Auto-unlock process is canceled after a user unlocks the lock from Apple Watch
+Auto-unlock process will be cancelled when you unlock the Lock from Apple Watch that is paired with iPhone that has active auto-unlock process. This feature will make sure that if user unlock his Lock during auto-unlock process, Lock will not be unlocked for second time when user is already in home. 
+
+## Additional hints for iOS 15.1 users who should upgrade the iOS system to use Auto-unlock feature.
+We know that some iOS versions have issues that can impact auto-unlock feature. If user is using iOS version have such an issue, in auto-unlock settings will be presented information that auto-unlock can work unstable and if we know that there is an update that fix this issue, this information will be also presented.\
```

Comment:
maybe "information will be presented"?

Reply:
or would be better.

If user is using iOS version with any detected issue, in auto-unlock settings we will show an information, that this feature can work unstable.

Reply:
or would be better.

If user is using iOS version with any detected issue, in auto-unlock settings we will show an information, that this feature can work unstable.